src/Renderer/c3DEngine.js

```diff
@@ -38,11 +38,13 @@ function c3DEngine(rendererOrDiv) {
         this.renderer.setSize(this.width, this.height);
     }.bind(this);
 
-    //
+    var canvas = document.createElement('canvas');
+    canvas.tabIndex = -1;
+    canvas.addEventListener('click', () => canvas.focus());
```

Comment:
I have a better idea: I think i'll add some options to define the focus policy in my FPControls instead. By default, it'll do nothing and let the client completely deal with focus, but it'll have a `focusOnClick` and `focusOnHover` options.